collector/monitors/configs_monitor.go

```diff
@@ -111,8 +107,13 @@ func (m *ConfigsCRC32Monitor) Handler(ctx context.Context) error {
 			m.logger.Errorf("failed to marshal %s: %+v", m.Name(), err)
 		}
 
-		m.metrics[metric].Set(float64(crc32.ChecksumIEEE(data)))
+		tmpMetricVectors[ConfigCRC32].Set(metric, float64(crc32.ChecksumIEEE(data)))
```

Comment:
> Set(metric

the `metric` variable is not actually a metric, it's a label, right? i think it would be nice to make this less ambiguous throughout the code